shell.nix

```diff
@@ -0,0 +1,22 @@
+with import <nixpkgs> { };
+
+let
+  moz_overlay = import (builtins.fetchTarball
+    "https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz");
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
+in mkShell rec {
+  name = "eww";
+
+  buildInputs = [
+    glib
+    cairo
+    atk
+    gdk-pixbuf
+    pango
+    gtk3
+    nixpkgs.latest.rustChannels.nightly.rust
+  ];
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+
+  RUST_SRC_PATH = "${pkgs.rust.packages.stable.rustPlatform.rustLibSrc}";
+}
```

Comment:
you won't sneak this in like that :<

Reply:
Oh no! That was an accident, i promise.

I ran git add -A, and since it wasn't added to the .gitignore it got added to the staged files.